client/app/components/adminAndModerator/ResultForm.tsx

```diff
@@ -296,6 +296,20 @@ const ResultForm = ({
             )}
           </div>
         )}
+        <div className="mt-2">
+          {round?.timeLimit && (
+            <div>
+            Time limit:&nbsp;
+              {getFormattedTime(round.timeLimit.centiseconds)} {round?.timeLimit.cumulativeRoundIds.length > 0 && 'in total'}
+            </div>
+          )}
+          {round?.cutoff && (
+            <div>
+              Cutoff:&nbsp;
+              {getFormattedTime(round.cutoff.attemptResult)}
+            </div>
+          )}
+        </div>
```

Comment:
I think the outer div isn't needed here, because if there is no time limit or cutoff, it would just remain in the markup, empty. Can you remove it and also make the time limit and cutoff show up on the same line, with the cutoff being optional? Keep in mind that there can't be a cutoff without a time limit also present.